docs/Changelog.md

```diff
@@ -10549,6 +10549,62 @@ This version of the operator has been available since version 11 of the default
 <dd>tensor of bool, which should be a scalar.</dd>
 </dl>
 
+### <a name="RoiCropAndResize-11"></a>**RoiCropAndResize-11**</a>
+
+  Extracts crops from the input image tensor and resizes them using bilinear sampling or nearest neighbor sampling
+  (possibly with aspect ratio change) to a common output size specified by crop_height and crop_width.
+  Returns a tensor with crops from the input image at positions defined at the bounding box locations in boxes.
+  The cropped boxes are all resized (with bilinear or nearest neighbor interpolation) to
+  a fixed size = [crop_height, crop_width]. The result is a 4-D tensor [num_boxes, crop_height, crop_width, depth].
+  The resizing is corner aligned. Suppose the input tensor X of shape (N, C, H, W), and the input rois tensor R of shape (M, 4),
+  and the batch indices tensor B of shape (M, ). The shape of output tensor Y is (M, C, CH, CW) where CH and CW is the crop_height
+  and crop_width. For each element Y[m, c, ch, cw], the image coordinate in the original X, (ch_x, cw_x),
+  is calculated at first, based on ch, cw, R[m, :], and B[m]. Then Y[m, c, ch, cw] = mode(X[B[m], c, ch_x, cw_x], extrapolation_value)
+  where mode is bilinear or nearest.
+  The equivalent TensorFlow version can be find [here](https://www.tensorflow.org/api_docs/python/tf/image/crop_and_resize).
+
+#### Version
+
+This version of the operator has been available since version 11 of the default ONNX operator set.
+
+#### Attributes
+
+<dl>
+<dt><tt>extrapolation_value</tt> : float (default is 0.0)</dt>
+<dd>Value used for extrapolation, when applicable. Default is 0.0f. </dd>
```

Comment:
Can you describe `extrapolation_value` in more details?

Reply:
This description here is same as tensorflow there. I can add "when the corresponding input coordinate is not available" (because of out of tensor bound) there.

Reply:
I know that the description match TF one, but it is lacking. Can you add more details?

Reply:
This [PR](https://github.com/onnx/onnx/pull/2057) will add crop to Resize op, so the extrapolation_value will be used there. The plan is that RoiCropAndResize will be a function of Resize. Does it make sense? I will hold this PR until Resize is finalized.